zokrates_js/src/lib.rs

```diff
@@ -21,82 +23,88 @@ pub struct CompilationResult {
     abi: String,
 }
 
+impl ResolverResult {
+    fn to_tuple(&self) -> (String, String) {
```

Comment:
I suspect this could be `into_tuple` and not perform any clone?

Reply:
yeah good catch